x/upgrade/keeper/keeper.go

```diff
@@ -48,6 +50,50 @@ func (k Keeper) SetUpgradeHandler(name string, upgradeHandler types.UpgradeHandl
 	k.upgradeHandlers[name] = upgradeHandler
 }
 
+func (k *Keeper) SetModuleManager(mm *module.Manager) *Keeper {
+	k.ModuleManager = mm
+	return k
```

Comment:
i don't think we generally return the caller in setters in the SDK

Reply:
Yeah, i saw some other module do this when handling references to keeper in simapp. removed in new commit